node/blockAPI/shardBlock_test.go

```diff
@@ -0,0 +1,304 @@
+package blockAPI
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/data/api"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/node/mock"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/testscommon"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShardAPIBlockProcessor_GetBlockByHash_InvalidHashShouldErr(t *testing.T) {
```

Comment:
Remove the second _ . Valid also for L55, L230

Reply:
removed